### DISCUSSION TITLE
Accessing detailed marketplace information

### DISCUSSION THREAD

**Discussion by bk1n** [joalla/discogs_client#60] [General] [answered]
Hi, 

I am very new to Python and to this discogs client (thank you for maintaining btw!) but am trying to build a Python script that will search for current marketplace listing for a release, filter them (e.g. media in NM/VG+ conditions and shipping from UK) and then look for changes regularly in the output.

I was hoping to use your API to search for marketplace info as above and then create my own script to parse out the required data and look for changes. I've tried to dig multiple layers into the release object:
``` 
print(release.marketplace_stats.num_for_sale)
print(release.marketplace_stats.lowest_price) 
```
but can only see this data. Am I missing something? Is it possible to find each specific listing of an item, the associated condition, shipping country, price, etc. for a release?

Thanks!

**Comment by AnssiAhola** [comment, marked as answer]
Hey, unfortunately it seems to be impossible to search the marketplace with the API.

Those `marketplace_stats` in a `Release` object only have those two fields `lowest_price`, `num_for_sale`
https://www.discogs.com/developers/#page:database,header:database-release

You can however get specific sellers listings https://www.discogs.com/developers/#page:marketplace,header:marketplace-inventory
if that helps at all. 😅
